internal/bundler/libfuzzer_bundler.go

```diff
@@ -511,6 +516,7 @@ depsLoop:
 	// Add dictionary to archive
 	var archiveDict string
 	if b.opts.Dictionary != "" {
+		log.Debug("Adding dictionary")
```

Comment:
Does it make sense to print the value of `b.opts.Dictionary` as well?

```suggestion
        log.Debugf("Adding dictionary ", b.opts.Dictionary)
```